### ISSUE TITLE
Raiden can't run in Goerli

### ISSUE THREAD

**Issue by LefterisJP** [raiden-network/raiden#5122]
Raiden fails to run in Goerli with the following error:

```
[TokenNetworkRegistry]Address 0x62436bbadB629498107278486C2A367FfdFEb2f3 has wrong code.. Please update your Raiden installation.
```

As seen [here](https://github.com/raiden-network/raiden/issues/5078#issuecomment-544377177)

**Comment by pirapira** [comment]
I think 0.33.2 is broken. https://github.com/raiden-network/raiden-contracts/issues/1298

**Comment by pirapira** [comment]
@LefterisJP how do I reproduce this issue locally?  I believe I have a fix https://github.com/raiden-network/raiden-contracts/pull/1299 so I want to try.

**Comment by karlb** [comment]
I've seen it in the scenarios, but just starting the Raiden client should be enough.

**Comment by LefterisJP** [comment]
@pirapira @karlb yes just start raiden using 0.33.2 contracts and goerli. It will happen immediately.

**Comment by pirapira** [comment]
Yes, I see the problem with:
```
git checkout cdf0143885264f7e67b54400b23b914f9c198052
make install-dev
raiden --eth-rpc-endpoint $GOERLI --network-id 5 --routing-mode private
```

**Comment by pirapira** [comment]
After installing `raiden-contracts` on this branch https://github.com/raiden-network/raiden-contracts/pull/1299

I don't see the problem, and the API starts.

**Comment by LefterisJP** [comment]
@pirapira since we run Raiden on Goerli now this can be closed, right?
